api/projects/models.py

```diff
@@ -194,6 +194,10 @@ def show_edge_identity_overrides_for_feature(self) -> bool:
             == IdentityOverridesV2MigrationStatus.COMPLETE
         )
 
+    @property
+    def get_org_id(self) -> int:
+        return self.organisation_id
```

Comment:
I don't understand why this exists and, if it does need to exist, why it's not named appropriately. `property`s should not have the `get_` prefix by convention. 

It should either be: 
 1. Removed entirely
 2. Changed to `org_id` if it needs to be a property
 3. Change to a method, not a property (and probably updated to `get_organisation_id` - I don't see the need to shorten `organisation` here)

Note: I _really_ don't like option 2 here because we're seemingly just reimplementing the existing `organisation_id` property provided by django. Let's try to go with option 1 or 3.

Reply:
I opted for the first option, this property has been removed, and the organisation ID is obtained directly.